workflow/libero_flow_poc/libero_flow/state_utils.py

```diff
@@ -0,0 +1,86 @@
+from typing import Dict
+
+import requests
+
+from libero_flow.conf import (
+    ACTIVITY_API_URL,
+    EVENT_API_URL,
+    WORKFLOW_API_URL,
+)
+
+
+WORKFLOW_DECISION_SCHEDULED = 'WorkflowDecisionScheduled'
```

Comment:
it seems `WorkflowDecisionStarted` and `WorkflowDecisionFinished` are missing, for completeness. Not necessarily to be exposed to the event bus, but part of the diagnostics here? (`DEBUG`-level logging for example)

Reply:
I was on the fence when implementing the events whether to add these. As the decider has a consistent algorithm unaffected by user-defined functions (like activities are), unless the actual decider fails, the output and execution time pretty static and predictable.

But as you say, these can just be filtered anyway so I will add them.